fiboa_cli/datasets/fs_za_ct.py

```diff
@@ -0,0 +1,144 @@
+URI = "/home/byteboogie/fieldscapes/south_africa/fs_za_ct.gpkg"
+
+# Unique identifier for the collection
+ID = "fs_za_ct"
+# Title of the collection
+TITLE = "Field boundaries for Cape Town, South Africa"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """ The dataset contains field boundaries for the Cape Town, South Africa."""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [20.521492384730347, -34.39922362572791, 21.04341451023305, -33.980506187460875]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "Planet, Radiant Earth Foundation, Western Cape Department of Agriculture, & German Aerospace Center (DLR)"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://beta.source.coop/esa/fusion-competition/"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "ESA Fusion Competition"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC BY-NC-SA 4.0"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {"title": "CC-BY-4.0", "href": "https://creativecommons.org/licenses/by/4.0/", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {
+    'fid': 'id',
+    'grid_id': 'grid_id',
+    "SHAPE_AREA": "area",
+    "SHAPE_LEN": "perimeter",
+    'geometry': 'geometry',
+    'crop_id': 'crop_id',
+    'crop_name': 'crop_name'
+}
+
+# Add columns with constant values.
+# The key is the column name, the value is a constant value that's used for all rows.
+ADD_COLUMNS = {
+    "determination_datetime": "2021-01-01T00:00:00Z"
+}
+
+# A list of implemented extension identifiers
+EXTENSIONS = []
+
+# Functions to migrate data in columns to match the fiboa specification.
+# Example: You have a column area_m in square meters and want to convert
+# to hectares as required for the area field in fiboa.
+# Function signature:
+#   func(column: pd.Series) -> pd.Series
+COLUMN_MIGRATIONS = {
+    "area_m": lambda column: column * 0.0001
+}
+
+# Filter columns to only include the ones that are relevant for the collection,
+# e.g. only rows that contain the word "agriculture" but not "forest" in the column "land_cover_type".
+# Lamda function accepts a Pandas Series and returns a Series or a Tuple with a Series and True to inverse the mask.
+COLUMN_FILTERS = {
+    
+}
+
+# Custom function to migrate the GeoDataFrame if the other options are not sufficient
+# This should be the last resort!
+# Function signature:
+#   func(gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame
+MIGRATION = None
+
+# Schemas for the fields that are not defined in fiboa
+# Keys must be the values from the COLUMNS dict, not the keys
+MISSING_SCHEMAS = {
```

Comment:
crop_id and crop_name seem to be missing?!

Reply:
Fixed the issue with the latest push